TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -423,9 +423,6 @@ TYPO3:
       'TYPO3.Neos:Preview':
         maximumWidth: 1000
         maximumHeight: 1000
-      'TYPO3.Neos:IconPreview':
-        maximumWidth: 16
-        maximumHeight: 16
```

Comment:
yes, IMO it's better to reuse the existing thumbnails (250x250) than generating separate icons for all of them.. there's a much higher chance those thumbnails exists already, ensuring that calls like this one and for link/reference searches don't waste time on generating thumbnails.. so the speed gain of smaller images is outweighed by fewer needed thumbnails, larger re-use and high probability of those thumbnails already existing. I didn't add it in the first place for the other places where small icons are used as well (media list view & link search)